run.py

```diff
@@ -158,6 +160,17 @@ def harvest_by_null(queue='priority-bumblebee',
     app.logger.info('Done submitting {0} pages.'.format(i))
     print i, last_id
 
+    
+def iscachable(target):
```

Comment:
if you make this part of the application (method), then various calls can provide/override this filter - just a thought